test/e2e/README.md

```diff
@@ -1,5 +1,6 @@
 # End-to-End Tests
 
-- [multi-cluster deployment](multi-cluster-deployment)
-- [singleton status return](singleton-status)
-- [Object cleaning](object-cleaning)
+- [all the end to end tests](ginkgo)
+- [multi-cluster deployment using shell scripts](multi-cluster-deployment)
+- [singleton status return using shell scripts](singleton-status)
+- [object fields filtering using shell scripts](object-cleaning)
```

Comment:
Looks like a rebase lost a change here. It should be "object cleaning" rather than "object fields filtering".

Reply:
fixed.